core/server/worker/src/main/java/alluxio/worker/block/UfsIOManager.java

```diff
@@ -45,6 +46,7 @@
  * Control UFS IO.
  */
 public class UfsIOManager implements Closeable {
+  private static final Logger LOG = LoggerFactory.getLogger(UfsIOManager.class);
```

Comment:
Is this used?

Reply:
used for many debugging so try to save it for future use